GitHubRepositories.ps1

```diff
@@ -1185,6 +1188,9 @@ filter Set-GitHubRepository
 
         [switch] $IsTemplate,
 
+        [ValidateSet('Enabled', 'Disabled')]
+        [string] $SecretScanning,
```

Comment:
My preference is to have all the switch parameters be at the end of the param set (with `$AccessToken` being the final one).  Given that, please move this to be before the first `[switch]` (as well as the `.PARAMETER` documentation as well).

Reply:
Done